scripts/update_bootlint.js

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env node
+
+/* eslint-env shelljs */
+
+'use strict';
+
+require('shelljs/global');
+var https       = require('https');
+var fs          = require('fs');
+var path        = require('path');
+
+var version     = process.argv[2];
+
+var basedir     = path.join(__dirname, '..');
+var bootlintDir = path.join(basedir, 'public', 'bootlint', version);
+var UGLIFYJS    = path.join(basedir, 'node_modules/.bin/uglifyjs');
+
+if (!version) {
+    echo('Valid Bootlint version required.');
+    process.exit(1);
+}
+
+// strip leading 'v' if present
+version = version.replace(/^v/, '$1');
+
+if (test('-d', bootlintDir)) {
+    echo('Bootlint version already found.');
+    process.exit(1);
+}
+
+mkdir(bootlintDir);
+pushd(bootlintDir);
+
+
+https.get('https://raw.githubusercontent.com/twbs/bootlint/v' + version + '/dist/browser/bootlint.js', function (res) {
+    var statusCode = res.statusCode;
+    var error = { };
+    var file = fs.createWriteStream(path.join(bootlintDir, 'bootlint.js'));
+
+    if (statusCode !== 200) {
+        error = new Error('Request Failed.\nStatus Code: ' + statusCode);
+    }
+
+    if (error) {
```

Comment:
I believe this would always return true, a few tests on the node console...

```node
jmervine@jmervine-ltm ~ $ node
> var error = {};
undefined
> !!(error);
true
> var error;
undefined
> !!(error);
true
> var error = undefined;
undefined
> !!(error);
false
```

`error` should be set to `undefined` on line 37.

```node
var error = undefined;
```